rules/expiring-todo-comments.js

```diff
@@ -47,7 +47,7 @@ const messages = {
 		'Unexpected \'{{matchedTerm}}\' comment without any conditions: \'{{comment}}\'.',
 };
 
-const packageResult = readPkgUp.sync();
+const packageResult = readPkgUp.sync({normalize: false});
```

Comment:
Can you add a comment to explain why we are doing this here?

Reply:
Ah, yes of course. Will do.

Reply:
Done.